PSPs/drafts/psp-6.md

```diff
@@ -0,0 +1,2903 @@
+      - [1.11.15.3. Example](#111153-example)
+    - [1.11.16. state_unsubscribeStorage (pubsub)](#11116-state_unsubscribestorage-pubsub)
+      - [1.11.16.1. Parameter](#111161-parameter)
+      - [1.11.16.2. Response](#111162-response)
+      - [1.11.16.3. Example](#111163-example)
+  - [1.12. Child State](#112-child-state)
+    - [1.12.1. childstate_getKeys](#1121-childstate_getkeys)
+      - [1.12.1.1. Parameter](#11211-parameter)
+      - [1.12.1.2. Response](#11212-response)
+    - [1.12.2. childstate_getStorage](#1122-childstate_getstorage)
+      - [1.12.2.1. Parameter](#11221-parameter)
+      - [1.12.2.2. Response](#11222-response)
+    - [1.12.3. childstate_getStorageHash](#1123-childstate_getstoragehash)
+      - [1.12.3.1. Parameter](#11231-parameter)
+      - [1.12.3.2. Response](#11232-response)
+    - [1.12.4. childstate_getStorageSize](#1124-childstate_getstoragesize)
+      - [1.12.4.1. Parameter](#11241-parameter)
+      - [1.12.4.2. Response](#11242-response)
+  - [1.13. Engine](#113-engine)
+    - [1.13.1. Engine Errors](#1131-engine-errors)
+    - [1.13.2. engine_createBlock](#1132-engine_createblock)
+    - [1.13.3. engine_finalizeBlock](#1133-engine_finalizeblock)
+  - [1.14. Payment](#114-payment)
+    - [1.14.1. Payment Errors](#1141-payment-errors)
+    - [1.14.2. payment_queryInfo](#1142-payment_queryinfo)
+  - [1.15. Contracts](#115-contracts)
+    - [1.15.1. Contracts Errors](#1151-contracts-errors)
+    - [1.15.2. contracts_call](#1152-contracts_call)
+    - [1.15.3. contracts_getStorage](#1153-contracts_getstorage)
+      - [1.15.3.1. Parameter](#11531-parameter)
+      - [1.15.3.2. Response](#11532-response)
+    - [1.15.4. contracts_rentProjection](#1154-contracts_rentprojection)
+  - [1.16. BABE](#116-babe)
+    - [1.16.1. BABE Errors](#1161-babe-errors)
+    - [1.16.2. babe_epochAuthorship](#1162-babe_epochauthorship)
+      - [1.16.2.1. Parameter](#11621-parameter)
+      - [1.16.2.2. Response](#11622-response)
+  - [1.17. Sync](#117-sync)
+    - [1.17.1. sync_state_genSyncSpec](#1171-sync_state_gensyncspec)
+      - [1.17.1.1. Parameter](#11711-parameter)
+      - [1.17.1.2. Response](#11712-response)
+      - [1.17.1.3. Example](#11713-example)
+  - [1.18. Copyright](#118-copyright)
+
+## 1.5. System
+
+System RPC API.
+
+### 1.5.1. System Errors
+
+RPC error codes are in the `2000` - `2999` range. No known error types are specified.
+
+### 1.5.2. system_name
+
+Get the node's implementation name.
+
+#### 1.5.2.1. Parameter
+
+None.
+
+#### 1.5.2.2. Response
+
+`STRING` - The node's name.
+
+#### 1.5.2.3. Example
+
+Request:
+
+```bash
+curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "system_name", "params":[]}' http://localhost:9933
+```
+
+Response:
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": "parity-polkadot",
+    "id": 1
+}
+```
+
+### 1.5.3. system_version
+
+Get the node implementation's version. Should be a semver string.
```

Comment:
Watch out for jargon like "semver"...if you use it, link out to a resource where the reader can learn more about it.